test/e2e/framework.go

```diff
@@ -976,7 +976,7 @@ func (data *TestData) createNginxPod(name, nodeName string) error {
 }
 
 // createServerPod creates a Pod that can listen to specified port and have named port set.
-func (data *TestData) createServerPod(name string, portName string, portNum int32, setHostPort bool) error {
+func (data *TestData) createServerPod(name string, nodeName string, portName string, portNum int32, setHostPort bool, hostNetwork bool) error {
```

Comment:
is nodeName used somewhere in the function?

Reply:
forgot to remove, thanks for the reminder!